modules/tinymce/CHANGELOG.md

```diff
@@ -27,6 +27,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Added a new `tablecolheader` toolbar button and menu item to toggle the header state of column cells #TINY-7482
 - Added a new `table-row-numbering` icon #TINY-7327
 - Added a new `iframe_aria_text` setting to set the iframe title attribute #TINY-1264
+- `editor.formatter.formatChanged` now supports filtering with format variables #TINY-7713
```

Comment:
It might just be me but this entry doesn't really explain what has been added

Reply:
Agreed and I'd also say `filtering` is the wrong term. It's more like listening to specific format vars changing.

Reply:
I've also just noticed this should be in `Improved` not `Added` as it's not really a new feature, it's an improvement to an existing feature.